x-pack/plugins/enterprise_search/public/applications/app_search/components/credentials/credentials_list/credentials_list.tsx

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { EuiBasicTable, EuiButtonIcon, EuiCopy } from '@elastic/eui';
+import { useActions, useValues } from 'kea';
+import { CriteriaWithPagination } from '@elastic/eui/src/components/basic_table/basic_table';
+
+import { i18n } from '@kbn/i18n';
+
+import {
+  CredentialsLogic,
+  ICredentialsLogicActions,
+  ICredentialsLogicValues,
+} from '../credentials_logic';
+import { IApiToken } from '../types';
+import { TOKEN_TYPE_DISPLAY_NAMES } from '../constants';
+import { apiTokenSort } from '../utils/api_token_sort';
+import { getModeDisplayText, getEnginesDisplayText } from '../utils';
+
+export const CredentialsList: React.FC = () => {
+  const { deleteApiKey, fetchCredentials, showCredentialsForm } = useActions(
+    CredentialsLogic
+  ) as ICredentialsLogicActions;
+
+  const { apiTokens, meta } = useValues(CredentialsLogic) as ICredentialsLogicValues;
+
+  const items = apiTokens.slice().sort(apiTokenSort);
+
+  const columns = [
+    {
+      name: 'Test',
+      width: '12%',
+      render: (item: IApiToken) => {
+        return <div>{item.name}</div>;
+      },
+    },
+    {
+      name: 'Type',
+      width: '18%',
+      render: (item: IApiToken) => {
+        return <div>{TOKEN_TYPE_DISPLAY_NAMES[item.type]}</div>;
+      },
+    },
+    {
+      name: 'Key',
+      width: '43%',
+      render: (item: IApiToken) => {
+        return (
+          <div>
+            <EuiCopy
+              textToCopy={item.key || ''}
+              afterMessage={i18n.translate('xpack.enterpriseSearch.appSearch.credentials.copied', {
+                defaultMessage: 'Copied',
+              })}
+            >
+              {(copy) => (
+                <div>
+                  <EuiButtonIcon
+                    onClick={copy}
+                    iconType="copyClipboard"
+                    aria-label={i18n.translate(
+                      'xpack.enterpriseSearch.appSearch.credentials.copyApiKey',
+                      {
+                        defaultMessage: 'Copy API Key to clipboard',
+                      }
+                    )}
+                  />
+                  {item.key}
+                </div>
+              )}
+            </EuiCopy>
+          </div>
+        );
+      },
+    },
+    {
+      name: 'Modes',
+      width: '10%',
+      render: (item: IApiToken) => {
+        return <div>{getModeDisplayText(item)}</div>;
+      },
+    },
+    {
+      name: 'Engines',
+      width: '10%',
```

Comment:
Would like to push for this being larger - my Engines column was pretty squashed for me:

```suggestion
      width: '18%',
```

### Before
<img width="1087" alt="" src="https://user-images.githubusercontent.com/549407/95274263-2e017d00-07fa-11eb-8305-a831359e6afe.png">

### After
<img width="1075" alt="" src="https://user-images.githubusercontent.com/549407/95274267-3063d700-07fa-11eb-8434-40b9fdc353ec.png">

Reply:
5d0ab10